src/test/scala/org/broadinstitute/dsde/workbench/sam/service/UserServiceSpec.scala

```diff
@@ -74,8 +74,8 @@ class UserServiceSpec extends FlatSpec with Matchers with TestSupport with Befor
     val mockGoogleDirectoryDAO = service.googleDirectoryDAO.asInstanceOf[MockGoogleDirectoryDAO]
     val groupEmail = WorkbenchGroupEmail(service.toProxyFromUser(defaultUserId.value))
     val allUsersGroupEmail = WorkbenchGroupEmail(service.toGoogleGroupName(UserService.allUsersGroupName.value))
-    mockGoogleDirectoryDAO.groups should contain key (groupEmail)
-    mockGoogleDirectoryDAO.groups(groupEmail) shouldBe Set(defaultUserEmail)
+//    mockGoogleDirectoryDAO.groups should contain key (groupEmail)
```

Comment:
How about using Mockito to verify that `CloudExtenstions.onUserCreate` was invoked? Somewhere in between not checking anything and leaving commented-out code (that you probably meant to remove anyway... but your hesitation makes me think you want to do _something_ here).